functions/controllers/user/imageUpload.js

```diff
@@ -0,0 +1,56 @@
+const Busboy = require('busboy')
+const path = require('path')
+const os = require('os');
+const fs = require('fs')
+const config = require('../../util/config')
+const { admin } = require('../../util/admin')
+const { status, message } = require('../../util/constants');
+const HttpStatus = require('http-status-codes');
+
+config.storageBucket
+
+const imageUpload = async (req, res, db) => {
+    const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } = HttpStatus
+    const { error, success } = status
+    const { wrongFileSubmitted, somethingWentWrong, imageUpdateSucces } = message
+
+    const busboy = new Busboy({headers: req.headers});
+    let imageFileName
+    let imageToBeUploaded = {};
+    const IMAGE_JPEG = 'image/jpeg'
+    const IMAGE_PNG = 'image/png'
+
+    await busboy.on('file', (fieldname, file, filename, encoding, mimetype) => {
+        if(mimetype !== IMAGE_JPEG && mimetype !== IMAGE_PNG){
+          return res.status(BAD_REQUEST).json({status: error, message: wrongFileSubmitted})
+        }
+        const imageExtension = filename.split('.')[filename.split('.').length - 1];
+        imageFileName  = `${Math.round(Math.random()*100000000)}.${imageExtension}`;
```

Comment:
keep number in a constant

Reply:
I think is okay here given that it is computed using the variable `imageExtension`